<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>玩家1准备</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LButton01">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton02">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1-2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton03">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1-3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton04">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1-4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton05">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1-5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton06">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton08">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2-5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton07">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2-3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton09">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton12">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3-5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3-2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3-4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton14">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4-3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton15">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4-5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton13">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton16">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton20">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5-5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton17">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5-2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton19">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5-4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton18">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5-3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton21">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton25">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6-5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton22">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6-2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton24">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6-4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LButton23">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6-3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton01">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>司</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton02">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>军</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton23">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>炸</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton24">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>炸</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton03">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>师</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton04">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>师</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton06">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>旅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton08">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>团</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton09">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>营</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton10">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>营</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton07">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>团</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton05">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>旅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton13">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>连</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton15">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>排</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton16">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>排</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton12">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>连</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton14">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>排</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton11">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>连</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton19">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton21">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>雷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton22">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>雷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton18">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton20">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>雷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton17">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>兵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton25">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>旗</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>340</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Reset_Button">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Confirm_Button">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Save_Button">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Open_Button">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Disp_Button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>准备布局</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionwan">
   <property name="text">
    <string>wan</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
